### PR TITLE
DM-35397: rename to ts_atmcssimulator and change " sim" to "-sim" in cscVersion

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -8,45 +8,48 @@ source:
...
     - LD_LIBRARY_PATH
     - OSPL_HOME
     - PYTHON_BUILD_VERSION
     - PYTHON_BUILD_LOCATION
-    - TS_CONFIG_ATTCS_DIR
     - LSST_DDS_DOMAIN
 
 test:
   requires:
+    - black
```
Comment (tribeiro) [inline conda/meta.yaml:25]: black is part of ts-conda-build, no need to add it here...

**Review by mareuter — APPROVED** [review]
One minor comment, but otherwise this looks fine to me.

File: conda/meta.yaml [hunk 2]
```diff
@@ -8,45 +8,47 @@ source:
...
+  build:
+    - python {{ python }}
     - setuptools
     - setuptools_scm
+    - ts-conda-build =0.3
+  run:
+    - python {{ python }}
+    - ts-idl
     - ts-salobj
     - ts-simactuators
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply conda/meta.yaml:53]: Which part are you referring to? it is not clear from your comment...

[the remark this answers is not in the document]
Comment (mareuter) [reply conda/meta.yaml:53]: The ts-simactuators package

[the remark this answers is not in the document]
Comment (tribeiro) [reply conda/meta.yaml:53]: oh! I see.. indeed.. I'm surprised it is not required for testing, it must be...

[the remark this answers is not in the document]
Comment (r-owen) [reply conda/meta.yaml:55]: Good catch (I think). I see that `ts-salobj` is not always listed with `{{ salobj_version }}`, as well (the amount of repetition in this file is appalling). I've taken a cleanup pass. Please have another look.

[the remark this answers is not in the document]
Comment (mareuter) [reply conda/meta.yaml:55]: Just to double check, ts-utils is required for test, but not for run. Is it pulled in the run section via an implicit dependency or is it not required?

**Pull request merged**